src/main/java/com/porpapps/wealthwatchapi/model/Portfolio.java

```diff
@@ -0,0 +1,17 @@
+package com.porpapps.wealthwatchapi.model;
+
+import lombok.Data;
+
+import java.time.LocalDate;
```

Comment:
You can remove this import now since you aren’t using LocalDate anymore

Reply:
done.